DESCRIPTION

```diff
@@ -50,7 +50,7 @@ Suggests:
     shiny,
     spelling,
     styler (>= 1.2.0),
-    testthat (>= 3.0.2)
+    testthat (>= 3.2.0)
```

Comment:
When I bump minimum version of testthat in `Suggests`, I think it makes sense to get totally current (meaning 3.2.1, today).
```suggestion
    testthat (>= 3.2.1)
```